rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -73,6 +74,19 @@ PPar.         Proc  ::= Proc "|" Proc1 ;
 
 separator Proc "," ;
 
+-- Let declarations
+DeclImpl.    Decl ::= [Name] NameRemainder "<-" [Proc] ;
+
+LinearDeclImpl.    LinearDecl ::= Decl ;
+separator nonempty LinearDecl ";" ;
+
+ConcDeclImpl.    ConcDecl ::= Decl ;
+separator nonempty ConcDecl "&" ;
+
+EmptyDeclImpl.      Decls ::= ;
+LinearDeclsImpl.    Decls ::= ";" [LinearDecl] ;
+ConcDeclsImpl.      Decls ::= "&" [ConcDecl] ;
```

Comment:
What is the difference between linear and concurrent let? 
My intuition is that with linear you can do `let a <- 0, b <- a-5` and with parallel it will throw error at compile time? Am I right here?

Reply:
The difference is simply that the expressions on the RHS of `<-` are evaluated concurrently. So `let x <- [1, 2].slice(0, 1) & y <- 2 + 3 in { Nil }` evaluates `[1, 2].slice(0, 1)` and `2 + 3` concurrently, then binds them to their respective variables.

Reply:
My question is more about if I can with linear let use variables from previous bindings, so is `let a <- 4, b <- a - 5` valid?

Reply:
Yes. Your expression translates to:

```
match 4 {
    a => {
        match a - 5 {
            b => 
        }
    } 
}
```